api/src/pcapi/routes/serialization/statistics_serialize.py

```diff
@@ -0,0 +1,24 @@
+import pydantic.v1 as pydantic_v1
+
+from pcapi.connectors.clickhouse.queries import YearlyAggregatedRevenueModel
+from pcapi.routes.serialization import BaseModel
+from pcapi.serialization.utils import to_camel
+
+
+class StatisticsQueryModel(BaseModel):
+    venue_ids: list[int] | int = []
+
+
+class StatisticsModel(BaseModel, YearlyAggregatedRevenueModel):
+    @pydantic_v1.root_validator(pre=True)
+    def non_empty_year(cls, values: dict) -> dict:
+        if not values.get("incomeByYear", {}).keys():
+            return values
+
+        for year in range(int(min(values["incomeByYear"].keys())), int(max(values["incomeByYear"].keys())) + 1):
+            if values["incomeByYear"].get(str(year)) is None:
+                values["incomeByYear"].update({str(year): {}})
+        return values
```

Comment:
[suggestion] Mini refacto pour plus de lisibilité

```suggestion
    @pydantic_v1.root_validator(pre=True)
    def non_empty_year(cls, values: dict) -> dict:
        income_by_year = values.get("incomeByYear", {})
        if not income_by_year.keys():
            return values

        min_year = int(min(income_by_year.keys()))
        max_year = int(max(income_by_year.keys()))

        for year in range(min_year, max_year + 1):
            if str(year) not in income_by_year:
                income_by_year[str(year)] = {}

        values["incomeByYear"] = income_by_year
        return values
```